coturn/coturn/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
```

Comment:
I believe the gateway opts will need an early access feature flag for this, correct?